pkgs/by-name/op/opensoldat/package.nix

```diff
@@ -84,13 +94,18 @@ stdenv.mkDerivation rec {
   # TODO(@sternenseemann): set proper rpath via cmake, so we don't need autoPatchelfHook
   runtimeDependencies = [ xorg.libX11 ];
 
+  patches = [
+    ./cmake_min_req.patch
+    ./build_gns.patch
+  ];
+
   # make sure soldat{,server} find their game archive,
   # let them write their state and configuration files
   # to $XDG_CONFIG_HOME/soldat/soldat{,server} unless
   # the user specifies otherwise.
   # TODO(@sternenseemann): rename config dir to opensoldat
   postInstall = ''
-    for p in $out/bin/soldatserver $out/bin/soldat; do
+    for p in $out/bin/opensoldatserver $out/bin/opensoldat; do
```

Comment:
Seems sensible to also rename `share/soldat` to match (which is an invention by us!).

Reply:
Good idea, done.